link-local-sdks.sh

```diff
@@ -3,19 +3,24 @@
 #       plz update this script so all devs can easily get setup
```

Comment:
add 
`set -e` 
to the top which will exit the script if any line throws an error